astropy/utils/tests/test_data.py

```diff
@@ -388,8 +388,10 @@ def test_download_with_sources_and_bogus_original(
         assert is_url_in_cache(u)
 
 
-@pytest.mark.skipif((3, 7) <= sys.version_info < (3, 8),
-                    reason="causes mystery segfault! possibly bug #10008")
+@pytest.mark.skipif((3, 7) <= sys.version_info < (3, 8) or
```

Comment:
I thought about removing `(3, 7) <=` but then remember we didn't bump minversion of Python to 3.7 until 4.2 and this PR is supposed to be backported to LTS.